### PR TITLE
Add stylesheet

### PR DESCRIPTION
**Pull request by jbence1994**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+    position: absolute;
```
Comment (pbkis0) [inline index.css]: Let's make it relative.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+}
```
Comment (pbkis0) [inline index.css]: Please add a new line

**Review by pbkis0 — CHANGES_REQUESTED** [review]
Please fix the comments

**Review by pbkis0 — APPROVED** [review]
LGTM

File: index.css (hunk 1 above)
Comment (jbence1994) [reply index.css]: fixed

File: index.css (hunk 2 above)
Comment (jbence1994) [reply index.css]: added

**Pull request merged**